addons/survey/views/survey_templates_statistics.xml

```diff
@@ -63,6 +63,8 @@
         <t t-set="comment_lines" t-value="question_data['comment_line_ids']"/>
         <t t-set="graph_data" t-value="question_data['graph_data']"/>
         <t t-set="table_data" t-value="question_data['table_data']"/>
+        <t t-set="single_table_line_height" t-value="35"/>
+        <t t-set="three_table_lines_height" t-value="83"/>
```

Comment:
It's a bit strange because 3 * 35 is not 83 :smile: 

If those are handcrafted pixel-perfect height, then maybe call it by question type, so "char_box_line_height" and "text_box_line_height" ?

EDIT: Or "default_line_height" and "text_box_line_height" as you seem to use the first one for other things than just char_box.